src/components/new-request/search-components/business-lookup-fetch.vue

```diff
@@ -62,7 +62,7 @@ export default class BusinessLookupFetch extends Mixins(CommonMixin, SearchMixin
         if (this.isConversion) {
           return this.isAlterable || 'This business cannot alter their business type'
         } else if (this.isRestoration) {
-          return this.isRestorable || 'This business cannot be restored'
+          return this.isRestorable || 'This business cannot be reactivated'
```

Comment:
Per Janis, this is to cover "restorations" and "reinstatements".

https://github.com/bcgov/entity/issues/18235#issuecomment-1771764387